renderlib/VolumeDimensions.cpp

```diff
@@ -187,6 +188,15 @@ VolumeDimensions::getChannelNames(const std::vector<uint32_t>& channels) const
   return channelNames;
 }
 
+std::string
+VolumeDimensions::sanitizeUnitsString(std::string units)
+{
+  // sanitize known weird characters
+  units = std::regex_replace(units, std::regex("µ"), "u");
+  units = std::regex_replace(units, std::regex("Å"), "A");
+  return units;
+}
+
```

Comment:
Do the font files not include these characters?

Reply:
Yes - the generated fonts are trying to capture a minimal set of characters to save texture space.   Ithink the API wants you to only use one contiguous range of character codes.  Maybe in a future PR I will figure out how to keep the nice µ character.